src/cascadia/TerminalApp/App.cpp

```diff
@@ -1165,6 +1166,20 @@ namespace winrt::TerminalApp::implementation
         }
     }
 
+    // Method Description:
+    // - Duplicates the current focused tab
+    void App::_DuplicateTabViewItem()
+    {
+        int focusedTabIndex = _GetFocusedTabIndex();

```

Comment:
Add a `const&` for all of these variable declarations.

Reply:
Thanks for the information with the tool. ;)
I added `const &` to the variable declarations.